### DISCUSSION TITLE
provide camera-streamer with H264 support

### DISCUSSION THREAD

**Discussion by fow0ryl** [MichaIng/DietPi#6560] [Software Request]
I have tested mjpegstreamer. Works but there is high network load when using higher resolutions.
On my (headless) NanoPi R1 I have installed [camera-streamer](https://github.com/ayufan/camera-streamer) and tested it as a replacement for mjpegstreamer. Works without any problems so far.
To get out the most benefit of camera-streamer the H264 stream of i.e. a USB camera should be used.

**Comment by MichaIng** [comment]
For reference, here an issue where the idea was mentioned to add an h264 output plugin to mjph-streamer: https://github.com/jacksonliam/mjpg-streamer/issues/336#ref-issue-1153206769

But camera-streamer seems to be a good alternative. Aside of Ayufan, also the OctoPrint maintainer is contributing.
